index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>Microverse Partnership: Ademola-Agasha</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+    <div class="container">
+        <div>
+            <header>
+                <h1 id="title" class="text-center">Ademola-Agasha Microverse Project
+                </h1>
+                <p id="description" class="text-center">we are software developers
+                </p>
+            </header>
+        </div>
+        <div>
```

Comment:
Use semantic HTML element ie `main` instead of `div`. `main` is more descriptive than `div` and it will represent well the main section of your webpage